src/LoRa.h

```diff
@@ -63,6 +63,7 @@ class LoRaClass : public Stream {
 #endif
   void idle();
   void sleep();
+  void continuousMode();
```

Comment:
@sabas1080 thanks for making the changes!

Things are looking very close to be ready to merge, two more notes:

1) Do you think this API should be private?

2) What do you thing about renaming this API to something like `LoRa.continuosReceiveMode()` or `LoRa.continuousRxMode()` for additional clarity?

Reply:
@sandeepmistry ok for me, I agree